src/Services/Air/templates/AIR_FARE_RULES_REQUEST.handlebars.js

```diff
@@ -2,21 +2,43 @@ module.exports = `
 <soapenv:Envelope xmlns:soapenv="http://schemas.xmlsoap.org/soap/envelope/">
     <soapenv:Header/>
     <soapenv:Body>
-        <air:AirFareRulesReq
-            TraceId="test" AuthorizedBy="{{requestId}}" TargetBranch="{{TargetBranch}}"
-            RetrieveProviderReservationDetails="false" FareRuleType="long"
+        <air:AirPriceReq
```

Comment:
this file is not used anywhere, it corresponds to templates.fareRules,
and we're using templates.price for lookupFareRules()

this is a template for booked reservations fare rules request from old code.